vendor/github.com/sirupsen/logrus/terminal_check_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build linux aix
+// +build linux aix solaris
```

Comment:
This change will fail the CI because it causes the file to become out of sync with the logrus repo.
It looks like the solaris build was fixed recently in logrus (https://github.com/sirupsen/logrus/commit/c1b61542d78094ef1a6fee0746528a251730141f), so maybe just updating the module to the latest will work.